src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -760,6 +760,8 @@ sub large_dither_checks {
     my $all_dither = shift;
     my $time_tol = 10;         # Commands must be within $time_tol of expectation
 
+    my $warn_free = 1;
```

Comment:
I don't see a good way to do all the tests and report if any failed without some new variable.  The other option would be to have a new array to store the errors and then have the individual checks push to that local array instead of the Obsid object warn array.